webapp/src/ts/services/transitions/test-user-for-contact.transition.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+import { Transition, Doc } from '@mm-services/transitions/transition';
+import { CreateUserForContactsService } from '@mm-services/create-user-for-contacts.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TestUserForContactsTransition extends Transition {
+  constructor(private createUserForContactsService: CreateUserForContactsService) {
+    super();
+  }
+
+  readonly name = 'create_user_for_contacts';
+  public readonly isEnabled = true;
+
+  init() {
+    return true;
+  }
+
+  filter(docs: Doc[]) {
+    if (!docs) {
+      return false;
+    }
+
+    return !!docs.filter(doc => doc?.create_user_for_contact === 'true').length;
```

Comment:
I think it is probably worth checking to see if the doc is actually a person contact: `this.contactTypesService.isPersonType(doc);`